AISKU/Initialization.ts

```diff
@@ -192,7 +191,14 @@ export class Initialization implements IApplicationInsights {
             identifier = identifier ? identifier : "ApplicationInsightsAnalytics";
```

Comment:
Lets get rid of the identifier parameter in this method, its making this complicated, and its not as if we can support a different config for a different identifier here.